client/me/account/main.jsx

```diff
@@ -566,7 +583,10 @@ const Account = React.createClass( {
 	 * These form fields are displayed when a username change is in progress.
 	 */
 	renderUsernameFields() {
-		const { translate, username, userSettings } = this.props;
+		const { translate, username } = this.props;
+
+		const isSaveButtonDisabled = ( this.getUserSetting( 'user_login' ) !== this.state.userLoginConfirm ) ||
```

Comment:
Another little nit here, but if we bump `this.state.submittingForm` to be checked first we can avoid calling the functions in the other two checks.

(Not a blocker since it's so, so minor)